systemtests/src/main/java/com/github/streamshub/systemtests/constants/TimeConstants.java

```diff
@@ -0,0 +1,25 @@
+package com.github.streamshub.systemtests.constants;
+
+import java.time.Duration;
+
+public class TimeConstants {
+    private TimeConstants() {}
+
+    private static long minutes(int minutes) {
+        return Duration.ofMinutes(minutes).toMillis();
+    }
+
+    private static long seconds(int seconds) {
+        return Duration.ofSeconds(seconds).toMillis();
+    }
```

Comment:
For clarity, I would name it like `minutesInMillis` and `secondsInMillis`, so you know what exactly the method is doing, if you really need this kind of encapsulation.